File: C#JAVA_Client_Server/ClientServcerC#Protocols/Solution1/Domain/Echipa.cs
using System;
using System.Collections.Generic;


namespace Domain
{
    [Serializable]
    public class Echipa : IObject<int>
    {
        String numeEchipa;

        public override string ToString()
        {
            return base.ToString()+" "+numeEchipa.ToString();
        }

        public Echipa(string numeEchipa)
        {
            this.numeEchipa = numeEchipa;
        }

        public Echipa()
        {
            
        }

        public string NumeEchipa
        {
            get => numeEchipa;
            set => numeEchipa = value;
        }
        
        public int Id { get; set; }
    }
}

File: C#JAVA_Client_Server/ClientServcerC#Protocols/LongExample/Domain/src/main/java/entities/domain/Echipa.java
package entities.domain;

public class Echipa extends Entitate<Integer> {
    String numeEchipa;

    public Echipa(String numeEchipa) {
        this.numeEchipa=numeEchipa;
    }

    @Override
    public String toString() {
        return numeEchipa;
    }

    public String getNumeEchipa() {
        return numeEchipa;
    }

    public void setNumeEchipa(String numeEchipa) {
        this.numeEchipa = numeEchipa;
    }


}
